lib/tasks/mfa_policy.rake

```diff
@@ -0,0 +1,32 @@
+require "resolv"
+
+def mx_exists?(email)
+  domain = email.split("@").last
+  mx_resolver = Resolv::DNS.new
+  mx_resolver.timeouts = 10
+
+  return false if mx_resolver.getresources(domain, Resolv::DNS::Resource::IN::MX).empty?
+  true
+rescue StandardError => e
+  puts "Error during processing: #{$ERROR_INFO}"
+  puts "Backtrace:\n\t#{e.backtrace.join("\n\t")}"
+  false
+end
```

Comment:
Since this is a self-contained rake task, I didn't DRY this up. I've simply copied it from the [mfa_notification rake task](https://github.com/rubygems/rubygems.org/blob/master/lib/tasks/mfa_notification.rake#L1-L14).